src/test/java/spoon/test/query_function/testclasses/VariableReferencesModelTest.java

```diff
@@ -26,7 +27,11 @@ public void localVarsInNestedBlocks() {
 				int field = 1;
 				assertTrue(field == 1);
 			}
-			int f1,f2,f3,field = 2,f4;

```

Comment:
limitation of sniper?

Reply:
Seems like I sadly

Reply:
Shall I revert it to the old style?

Reply:
nope, that's a good trace of the problem, see https://github.com/INRIA/spoon/issues/3386